tests/e2e/utils/workspace/TestWorkspaceUtil.ts

```diff
@@ -29,17 +30,18 @@ export class TestWorkspaceUtil implements ITestWorkspaceUtil {
     readonly polling: number = TestConstants.TS_SELENIUM_WORKSPACE_STATUS_POLLING;
     constructor(
         @inject(CLASSES.DriverHelper) private readonly driverHelper: DriverHelper,
-        @inject(CLASSES.CheApiRequestHandler) private readonly processRequestHandler: CheApiRequestHandler
+        @inject(CLASSES.CheApiRequestHandler) private readonly processRequestHandler: CheApiRequestHandler,
+        @inject(CLASSES.ApiUrlResolver) private readonly apiUrlResolver: ApiUrlResolver
     ) { }
 
     public async waitWorkspaceStatus(namespace: string, workspaceName: string, expectedWorkspaceStatus: WorkspaceStatus) {
```

Comment:
please add deprecation notice into ITestWorkspaceUtil.ts of methods which were only supported on CheServer implementation, unless they are being completely removed:
```
    /**
     * @deprecated Method deprecated. Works with CHE server only
     */
```